test/TestProjects/MgmtMockAndSample/src/Generated/VirtualMachineExtensionImageCollection.cs

```diff
@@ -197,7 +197,7 @@ public virtual Pageable<VirtualMachineExtensionImageResource> GetAll(Cancellatio
         /// </summary>
         /// <param name="type"> The String to use. </param>
```

Comment:
maybe this will never happen in real SDKs because we have linter rules to make sure every parameter/property should have a description, but this should be `string` instead of the capitalized `String`?

Reply:
Yes, and this is specifically disabled for `String/string` pair because it affects several hundreds  of files.